lib/src/install/config.rs

```diff
@@ -319,3 +345,189 @@ block = ["tpm2-luks"]"##,
     // And verify passing a disallowed config is an error
     assert!(install.get_block_setup(Some(BlockSetup::Direct)).is_err());
 }
+
+#[test]
+/// Verify that kargs are only applied to supported architectures
+fn test_arch() {
```

Comment:
Can we split up this PR into two?  The install-time karg architecture matching is logically distinct from "day 2 changeable" kargs I think, and can go in separately.

Reply:
https://github.com/containers/bootc/pull/567/files